liblouis/compileTranslationTable.c

```diff
@@ -1092,8 +1112,36 @@ getNextAttribute(TranslationTableHeader *table) {
 
 static CharacterClass *
 addCharacterClass(const FileInfo *file, const widechar *name, int length,
-		TranslationTableHeader *table) {
+		TranslationTableHeader *table, int validate) {
 	/* Define a character class, Whether predefined or user-defined */
+	if (validate) {
+		for (int i = 0; i < length; i++) {
+			if (!((name[i] >= 'a' && name[i] <= 'z') ||
+						(name[i] >= 'A' && name[i] <= 'Z'))) {
+				// don't abort because in some cases (before/after rules)
+				// this will work fine, but it will not work in multipass
+				// expressions
```

Comment:
This is weird but I think I understand why

Reply:
I added https://github.com/liblouis/liblouis/issues/1133